src/app/zap-templates/templates/app/helper.js

```diff
@@ -109,7 +109,7 @@ var endpointClusterWithInit = [
   'Thermostat',
 ];
 var endpointClusterWithAttributeChanged = [ 'Identify', 'Door Lock', 'Pump Configuration and Control' ];
-var endpointClusterWithPreAttribute     = [ 'IAS Zone', 'Thermostat User Interface Configuration' ];
+var endpointClusterWithPreAttribute     = [ 'IAS Zone', 'Door Lock', 'Thermostat User Interface Configuration' ];
```

Comment:
This is weird to see this helper code mixing thing, but out of scope

Reply:
This was done to enable pre-attribute change callback for Door Lock cluster as it it not generated by default.